js/privatebin.js

```diff
@@ -3917,10 +3943,12 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
             try {
                 // decrypt attachments
                 if (paste.attachment) {
-                    // try to decrypt paste and if it fails (because the password is
-                    // missing) return to let JS continue and wait for user
-                    if (!decryptAttachment(paste, key, password)) {
-                        return;
+                    if (AttachmentViewer.$attachment.length) {
```

Comment:
If you are in AttachmentViewer, you can use `me`. Otherwise this should not work as variables should not be exposed publicly in modules. Create a getter method if needed.